api/v1/models/kube_proxy_replacement.go

```diff
@@ -43,6 +43,9 @@ type KubeProxyReplacement struct {
 	// mode
 	// Enum: [Disabled Strict Probe Partial]
 	Mode string `json:"mode,omitempty"`
+
+	// flag bpf-lb-sock-hostns-only
+	BPFSocketLBHostnsOnly bool `json:"bpfSocketLBHostnsOnly,omitempty"`
```

Comment:
This should go under `KubeProxyReplacementFeatures` (see above L 41). Please take a look at how other flags are exposed.

Reply:
This is still not addressed. You've added the flag under `KubeProxyReplacement`, and not `KubeProxyReplacementFeatures`. Was there an issue in adding it under the latter?